### PR TITLE
Adds a ghost verb to see all available ghost spawners

### PR DESCRIPTION
**Pull request by lzimann**
:cl: Lzimann
add: Ghosts now have a way to see all available ghost roles! Check your ghost tab!
/:cl:
interface image

Nothing too fancy, these could probably get a real description in the future, since they are currently using the flavour text.

### CODE REVIEW

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -51,11 +51,12 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	// Used for displaying in ghost chat, without changing the actual name
 	// of the mob
 	var/deadchat_name
+	var/datum/spawners_menu/spawners_menu
 
 /mob/dead/observer/Initialize()
 	set_invisibility(GLOB.observer_default_invisibility)
 
-	verbs += /mob/dead/observer/proc/dead_tele
+	verbs += list(/mob/dead/observer/proc/dead_tele, /mob/dead/observer/proc/open_spawners_menu)
```
Comment (RemieRichards) [inline code/modules/mob/dead/observer/observer.dm]: lay it out on separate lines

**Comment by Tacolizard** [comment]
imo the first letter of each description should not be given a large font like they are in the screenshot. It might work for calligraphy or ancient manuscripts but it looks out of place in spess.

**Comment by lzimann** [comment]
this is from the flavour text, not my doing

**Comment by BeeSting12** [comment]
I like this. For some reason I read the title and thought you added the ghost verb to ghost spawns.

**Comment by lzimann** [comment]
I'm bad with titles

**Pull request merged**